core/domain/services/graphql/connections/RootQueryFormSectionsConnection.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\connections;
+
+use EEM_Form_Section;
+use EventEspresso\core\domain\services\graphql\connection_resolvers\FormSectionConnectionResolver;
+use EventEspresso\core\domain\services\graphql\abstracts\AbstractRootQueryConnection;
+use Exception;
+
+/**
+ * Class RootQueryFormSectionsConnection
+ * Description
+ *
+ * @package EventEspresso\core\domain\services\graphql\connections
+ * @author  Manzoor Wani
+ * @since   $VID:$
+ */
+class RootQueryFormSectionsConnection extends AbstractRootQueryConnection
+{
+
+
+    /**
+     * FormSectionConnection constructor.
+     *
+     * @param EEM_Form_Section               $model
+     */
+    public function __construct(EEM_Form_Section $model)
+    {
+        $this->model = $model;
+    }
+
+
+    /**
+     * @return array
+     * @since $VID:$
+     */
+    public function config()
+    {
+        return [
+            'fromType'           => 'RootQuery',
+            'toType'             => $this->namespace . 'FormSection',
+            'fromFieldName'      => lcfirst($this->namespace) . 'FormSections',
+            'connectionTypeName' => "{$this->namespace}RootQueryFormSectionsConnection",
+            'resolve'            => [$this, 'resolveConnection'],
+        ];
+    }
```

Comment:
we can also use PHP 7 return types in the function declarations, **IF** the method does not override a parent function that is not declared the same way.  ex:

```php
public function config(): array
```

is better, but NOT if this method is declared on a parent class (or in an interface) like:

```php
public function config()
```

because then the declarations don't match and hilarity will ensue in the form of exceptions :\

Reply:
Yeah, I am not used to strict types in PHP yet. Need to start using those 😄